source/_components/cover.rpi_gpio.markdown

```diff
@@ -27,6 +27,8 @@ To enable Raspberry Pi Covers in your installation, add the following to your `c
 # Example configuration.yaml entry
 cover:
   platform: rpi_gpio
+  relay_time: 0.2
+  state_pull_mode: 'UP'
```

Comment:
Those are optional entries. As we try to keep the configuration sample minimal, please add them only to the "Configuration variables".